packages/data-service/src/instance-detail-helper.ts

```diff
@@ -134,9 +141,22 @@ export async function getInstance(
     dataLake: buildDataLakeInfo(buildInfoResult),
     featureCompatibilityVersion:
       getParameterResult?.featureCompatibilityVersion.version ?? null,
+    isAtlas: checkIsAtlas(client, atlasVersionResult),
   };
 }
 
+function checkIsAtlas(
+  client: MongoClient,
+  atlasVersionInfo: AtlasVersionInfo
+): boolean {
+  const firstHost = client.options.hosts[0]?.host || '';
+
+  if (undefined === atlasVersionInfo.version) {
```

Comment:
non really important but seems that in the rest of the codebase we do `something === undefined`, if you want to keep it consistent